### PR TITLE
Don't fail log publishing when test results folder doesn't exist

### PR DESCRIPTION
**Pull request by ViktorHofer**

### CODE REVIEW

File: eng/pipelines/templates/jobs/vmr-build.yml [hunk 1]
```diff
@@ -451,10 +451,13 @@ jobs:
         cd "$(sourcesPath)"
         find artifacts/log/ -type f -name "*.binlog" -exec rsync -R {} -t ${targetFolder} \;
         find artifacts/log/ -type f -name "*.log" -exec rsync -R {} -t ${targetFolder} \;
-        find artifacts/TestResults/ -type f -name "*.binlog" -exec rsync -R {} -t ${targetFolder} \;
-        find artifacts/TestResults/ -type f -name "*.diff" -exec rsync -R {} -t ${targetFolder} \;
-        find artifacts/TestResults/ -type f -name "Updated*.txt" -exec rsync -R {} -t ${targetFolder} \;
-        find artifacts/TestResults/ -type f -name "*.trx" -exec rsync -R {} -t ${targetFolder} \;
+
+        if [ -n "$(ls -A 'artifacts/TestResults/')" ]; then
```
[the remark this answers is not in the document]
Comment (ViktorHofer) [reply eng/pipelines/templates/jobs/vmr-build.yml:455]: Yeah I considered that but as there was already a similar check below I went with this but you are right that it here probably makes more sense to just condition on the existence of the folder.

**Review by premun — APPROVED** [review]

**Pull request merged**